osquery/database/database.cpp

```diff
@@ -623,9 +617,6 @@ Status upgradeDatabase(int to_version) {
       return Status(1, "Database migration failed.");
```

Comment:
this status is wrong... should be something like 'DB migration ok, failed to set new version'

Reply:
Also above on line 607, we should LOG(ERROR) when `!migrate_status.ok()`